src/OpenTelemetry.ClrProfiler.Native/dd_profiler_constants.h

```diff
@@ -56,23 +69,35 @@ namespace trace {
     WStr("System.Text"),
     WStr("System.Threading"),
     WStr("System.Xml"),
-    WStr("Newtonsoft"),
-  };
+};
 
-  inline WSTRING skip_assemblies[]{
-      WStr("mscorlib"),
-      WStr("netstandard"),
-      WStr("System.Configuration"),
-      WStr("Microsoft.AspNetCore.Razor.Language"),
-      WStr("Microsoft.AspNetCore.Mvc.RazorPages"),
-      WStr("Anonymously Hosted DynamicMethods Assembly"),
-      WStr("ISymWrapper")
-  };
+const WSTRING skip_assemblies[]{WStr("mscorlib"),
+                                WStr("netstandard"),
+                                WStr("System.Configuration"),
+                                WStr("Microsoft.AspNetCore.Razor.Language"),
+                                WStr("Microsoft.AspNetCore.Mvc.RazorPages"),
+                                WStr("Anonymously Hosted DynamicMethods Assembly"),
+                                WStr("ISymWrapper")};
 
-  inline WSTRING managed_profiler_full_assembly_version = WStr("OpenTelemetry.ClrProfiler.Managed");
+const WSTRING mscorlib_assemblyName = WStr("mscorlib");
+const WSTRING system_private_corelib_assemblyName = WStr("System.Private.CoreLib");
+const WSTRING opentelemetry_clrprofiler_managed_loader_assemblyName = WStr("OpenTelemetry.ClrProfiler.Managed.Loader");
 
-  inline WSTRING calltarget_modification_action = WStr("CallTargetModification");
+const WSTRING microsoft_aspnetcore_hosting_assemblyName = WStr("Microsoft.AspNetCore.Hosting");
+const WSTRING dapper_assemblyName = WStr("Dapper");
 
-}  // namespace trace
+const WSTRING managed_profiler_full_assembly_version =
+    WStr("OpenTelemetry.ClrProfiler.Managed, Version=0.0.1.0, Culture=neutral, PublicKeyToken=34b8972644a12429");
 
-#endif  // DD_PROFILER_CONSTANTS_H
\ No newline at end of file
+const WSTRING managed_profiler_name = WStr("OpenTelemetry.ClrProfiler.Managed");
+
+const WSTRING nonwindows_nativemethods_type = WStr("OpenTelemetry.ClrProfiler.Managed.NativeMethods+NonWindows");
+
+// TODO: Enable if defined
+// const WSTRING appsec_nonwindows_nativemethods_type = WStr("OpenTelemetry.ClrProfiler.Managed.AppSec.Waf.NativeBindings.NativeLibrary+NonWindows");
```

Comment:
@zacharycmontoya Could you point a background story for that? Do we need it in OTel also?
Currently I have left out all of the managed classes connected to this feature.

cc @pjanotti @pellared

Reply:
So according to the SIG we don't need it in OTel.